src/domain/profile/Profile.tsx

```diff
@@ -35,12 +31,8 @@ const Profile: FunctionComponent = () => {
       </PageWrapper>
     );
   }
-  // userHasProfile selector saves us when Apollo cache gives a wrong answer.
```

Comment:
I managed to enter a weird state when pressing the back button right after submitting the first page form and getting logged in.

Reply:
But this logic is a bit wrong, gotta fix it another way.
Can you make step by step to reproduce?

Reply:
data that fetched should always be the single source of truth, not cached one

Reply:
Steps to reproduce - and it might be that this is unrelated to this change:

1. delete your profile
2. Submit front page form (so that you're logged in again)
3. press back button
4. try to continue with registration

Reply:
Based on the superficial testing we did, I think we can mark this one as resolved - and I still think it makes sense to look for any issues on the profile page. Hopefully the error doesn't happen after removing the normalizeProfileData function.